### PR TITLE
Update docs for log rotation

### PR DESCRIPTION
**Pull request by a-mccarthy**
Signed-off-by: Abigail McCarthy <mabigail@vmware.com>

related to: https://github.com/goharbor/harbor/issues/17074

### CODE REVIEW

**Comment by stonezdj** [comment]
> he types of operations forwarded? So, if you only had pull operations selected in your purge configuration, would Harbor honor that when skipping the database?

All operations are forwarded,  the select operations only apply for purge, no related to the forward or skipping the database.

**Review by stonezdj — APPROVED** [review]
LGTM

File: docs/administration/garbage-collection/_index.md [hunk 1]
```diff
@@ -38,10 +36,10 @@ To avoid triggering the garbage collection process too frequently, the availabil
     * **Daily**: Run garbage collection at midnight every day.
     * **Weekly**: Run garbage collection at midnight every Saturday.
     * **Custom**: Run garbage collection according to a `cron` job.
-    
-1. To delete untagged artifacts, select the check box **Delete Untagged Artifacts**.     
+
+1. To **Allow garbage collection on untagged artifacts**, select the option. If you have this option turned on, the next time Garbage collect runs on your Harbor instance, Harbor will delete untagged artifacts and then perform garbage collection on them.     
 1. Click **Save**.
-1. Select the **History** tab to view records of the 10 most recent garbage collection runs.
+1. View the 10 most recent garbage collection runs in the **GC History** table.
```
Comment (AllForNothing) [inline docs/administration/garbage-collection/_index.md:42]: now, it's 5, not 10.

Comment (a-mccarthy) [reply docs/administration/garbage-collection/_index.md]: Thanks! I updated this to be more generic so we dont have to worry about specific numbers if this changes in the future!

**Comment by a-mccarthy** [comment]
@AllForNothing I've updated this PR to include information about stopping log rotation and garbage collection. Please take a look at my changes and let me know if the are 👍 

related to: https://github.com/goharbor/harbor/issues/17102

**Review by AllForNothing — APPROVED** [review]
LGTM

**Pull request merged**